jekyll/_cci2/security.md

```diff
@@ -119,6 +104,20 @@ Following are the system events that are logged. See `action` in the Field secti
 - **success:** A flag to indicate if the action was successful.
 - **request:** If this event was triggered by an external request this data will be populated and may be used to connect events that originate from the same external request. The format is a JSON blob containing `id` (the request ID assigned to this request by CircleCI), `ip_address` (the original IP address in IPV4 dotted notation from which the request was made, eg. 127.0.0.1), and `client_trace_id` (the client trace ID header, if present, from the 'X-Client-Trace-Id' HTTP header of the original request).
 
+## Checklist To Using CircleCI Securely as a Customer
+
+If you are getting started with CircleCI there are some things you can ask your team to consider for security best practices as _users_ of CircleCI:
+
+- Minimise the number of secrets (private keys / environment variables) your
+  build needs and rotate secrets regularly.
+- Ensure the secrets you _do_ use are of limited scope - with only enough permissions for the purposes of your build.
```

Comment:
Likewise here with the scoping of the builds, I think the smallest amount of further explanation should be included. E.g., machine users on GH, not using personal/root AWS credentials

Reply:
@taxonomic-blackfish Thanks for pointing this out. Can you expand what you mean by "Machine user's on GH?" I'm not sure what security aspect that refers to.

How does this sound:

```
  - Ensure the secrets you _do_ use are of limited scope - with only enough permissions for the purposes of your build. Consider carefully adjudicating the role and permission systems of other platforms you use outside of CircleCI; for example, when using something such as IAM permissions on AWS, you should ensure that you properly assign authorization levels on a per user basis.
```

Reply:
```suggestion
- Ensure the secrets you _do_ use are of limited scope - with only enough permissions for the purposes of your build.
```

Reply:
Machine users: https://developer.github.com/v3/guides/managing-deploy-keys/#machine-users

I think that is an improvement, yes.